src/data/scripts/add_context_by_property_type.py

```diff
@@ -28,54 +30,24 @@
     'SiteEUI',
 ]
 
-# Columns that should be strings because they are immutable identifiers
-string_cols = [
```

Comment:
This (and `int_cols`) was duplicated in a lot of places, so I made it a central constant that can be reused.